debian/gen-flat-repo-fixtures.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -eux
+
+sudo apt-get install dpkg-dev apache2 dpkg-sig
```

Comment:
This is run in the (transient) container as root.

```suggestion
apt-get install dpkg-dev apache2 dpkg-sig
```

Reply:
Thank you for your comment! Changes have been integrated.